### ISSUE TITLE
Doesn't launch game

### ISSUE THREAD

**Issue by Wolf1030** [soulsmods/ModEngine2#42]
Since you closed my ticket even though it wasn't resolved. 

You said to place the files in the main folder with the eldenring.exe and I did, here's a screenshot, it doesn't work.

Since you also said:

>>The TOML file doesn't contain game paths, There's currently only 2 mechanisms for finding the game exe (and folder):

>Find an executable named <game.exe> adjacent to modengine2_launcher.exe

Sorry to say, but it does not.
https://puu.sh/ITM7j/c9881f74e3.png

**Comment by BDInthere** [comment]
> 

are you a notebook user?cause I cannot launch game with the .bat file on my notebook，and I have no idea what is going on 🤔

**Comment by Wolf1030** [comment]
> > 
> 
> are you a notebook user?cause I cannot launch game with the .bat file on my notebook，and I have no idea what is going on 🤔

nope. This mod manager. The other suggestion is to run the modmanager.exe if the bat isn't working, but both do not work for me. The "other" suggestion is to place all the files in the game's main folder then try to run either the .bat or .exe which this method doesnt work for me either.

**Comment by Wolf1030** [comment]
Alright, I feel dumb, but just to still give some info.

Normal launch of bat and exe did nothing.
Run as Admin bat did nothing, but run as Admin on the exe did. 

Still something to look into regardless  as to the bat isn't working. 

feel free to close it now.

**Comment by BDInthere** [comment]
Well here is the promble,
while I lauch the game with .bat I can see that the game is lauched in my task manager.
However,the game quit next moment.
And here's the log

**Comment by BDInthere** [comment]
[2022-04-11 xx:46:31.758] [modengine] [info] ModEngine version 2.0.0-94b783b0c497f6515d9783da22ca230520d174f7 initializing for Elden Ring
[2022-04-11 xx:46:31.758] [modengine] [info] Local settings loaded: true, Global settings loaded: false
[2022-04-11 xx:46:31.758] [modengine] [info] Main thread ID: 18684
[2022-04-11 xx:46:31.762] [modengine] [info] Enabling extension base
[2022-04-11 xx:46:31.762] [modengine] [info] Enabled extension base
[2022-04-11 xx:46:31.762] [modengine] [info] Enabling extension mod_loader
[2022-04-11 xx:46:31.762] [modengine] [info] Installing mod location mod
[2022-04-11 xx:46:31.762] [modengine] [info] Resolved mod path to E:\steam\steamapps\common\ELDEN RING\Game\mod
[2022-04-11 xx:46:31.763] [modengine] [info] Enabled extension mod_loader
[2022-04-11 xx:46:31.763] [modengine] [info] Enabling extension scylla_hide
[2022-04-11 xx:46:31.763] [modengine] [info] Found valid ScyllaHide injector. Injecting into game.
[2022-04-11 xx:46:31.763] [modengine] [info] Starting ScyllaHide injector: E:\steam\steamapps\common\ELDEN RING\Game\modengine2\tools\scyllahide\InjectorCLIx64.exe pid:13780 ./HookLibraryx64.dll nowait
[2022-04-11 xx:46:31.803] [modengine] [info] Launched Scyllahide process.
[2022-04-11 xx:46:32.057] [modengine] [info] ScyllaHide successfully injected
[2022-04-11 xx:46:32.057] [modengine] [info] Enabled extension scylla_hide
[2022-04-11 xx:46:32.140] [modengine] [info] Applied 3 hooks
[2022-04-11 xx:46:32.143] [modengine] [info] Starting worker thread

**Comment by jonathan-wenek** [comment]
I am having the same issue. Here is the log that it produced.

`[2022-04-12 14:27:39.440] [modengine] [info] ModEngine version 2.0.0-preview3-2144572dce7257d2cc477db78036e70dcd136b5a initializing for Elden Ring
[2022-04-12 14:27:39.440] [modengine] [info] Local settings loaded: true, Global settings loaded: false
[2022-04-12 14:27:39.440] [modengine] [info] Main thread ID: 2008
[2022-04-12 14:27:39.443] [modengine] [info] Enabling extension base
[2022-04-12 14:27:39.443] [modengine] [info] Enabled extension base
[2022-04-12 14:27:39.443] [modengine] [info] Enabling extension mod_loader
[2022-04-12 14:27:39.443] [modengine] [info] Installing mod location mod
[2022-04-12 14:27:39.443] [modengine] [info] Resolved mod path to C:\Program Files (x86)\Steam\steamapps\common\ELDEN RING\Game\ModEngine-2.0.0-preview3-win64\mod
[2022-04-12 14:27:39.443] [modengine] [info] Enabled extension mod_loader
[2022-04-12 14:27:39.516] [modengine] [info] Applied 3 hooks
[2022-04-12 14:27:39.518] [modengine] [info] Starting worker thread
[2022-04-12 14:27:39.622] [modengine] [info] Destroying
[2022-04-12 14:27:39.622] [modengine] [info] Destroying
[2022-04-12 14:27:39.622] [modengine] [info] Destroying
[2022-04-12 14:27:39.622] [modengine] [info] Destroying
[2022-04-12 14:27:39.622] [modengine] [info] Destroying
[2022-04-12 14:27:39.622] [modengine] [info] Destroying
[2022-04-12 14:27:39.622] [modengine] [info] Destroying
[2022-04-12 14:27:39.622] [modengine] [info] Destroying
[2022-04-12 14:27:39.622] [modengine] [info] Destroying
[2022-04-12 14:27:39.622] [modengine] [info] Destroying
`

**Comment by Kolchefen** [comment]
@Wolf1030 So the fix to this is to have steam launched before launching the .bat file.

**Comment by lhasxl** [comment]
i have the same problem exe does nothing as an admin nor the bat file

**Comment by ZellsAlpha** [comment]
doesnt work just auto closes, opening it with cmd says [2022-04-17 19:14:19.907] [stderr] [error] Couldn't create process: 10b

**Comment by Mo0nsterCat** [comment]
![image](https://user-images.githubusercontent.com/104241228/164866486-4af8f3a5-e6d1-415d-ac48-640b322fb7bb.png)
Check this if you use windows defender

**Comment by F-annaray** [comment]
thx for responding, but ehh  I do not speak Russian and cannot copy text to
put in Google  translation
Greetz
Kyrill

**Comment by TomVige** [comment]
Same unsolved problem

**Comment by F-annaray** [comment]
Can I have a response plz?

Regards
Kyrill

**Comment by F-annaray** [comment]
Got the response, try it as soon as possible 👍🙂👍

**Comment by jackywong0036** [comment]
`%~dp0\modengine2_launcher.exe -t er -c %~dp0\config_eldenring.toml`
replace "launchmod_eldenring.bat" and run as administrator

<hr>

if windows defender is enabled
switch off "Controlled folder access"
or whitelist "eldenring.exe"
https://docs.microsoft.com/en-us/microsoft-365/security/defender-endpoint/customize-controlled-folders?view=o365-worldwide

**Comment by kalip30** [comment]
> Since you closed my ticket even though it wasn't resolved.
> 
> You said to place the files in the main folder with the eldenring.exe and I did, here's a screenshot, it doesn't work.
> 
> Since you also said:
> 
> > > The TOML file doesn't contain game paths, There's currently only 2 mechanisms for finding the game exe (and folder):
> 
> > Find an executable named <game.exe> adjacent to modengine2_launcher.exe
> 
> Sorry to say, but it does not. https://puu.sh/ITM7j/c9881f74e3.png

Hey sorry to bother you but it looks like you have a cracked version if im right.. i do so too. How did you managed to run mod engine on cracked version? i cant make it work whatever i do.

**Comment by Friendlyx3** [comment]
> > Since you closed my ticket even though it wasn't resolved.
> > You said to place the files in the main folder with the eldenring.exe and I did, here's a screenshot, it doesn't work.
> > Since you also said:
> > > > The TOML file doesn't contain game paths, There's currently only 2 mechanisms for finding the game exe (and folder):
> > 
> > 
> > > Find an executable named <game.exe> adjacent to modengine2_launcher.exe
> > 
> > 
> > Sorry to say, but it does not. https://puu.sh/ITM7j/c9881f74e3.png
> 
> Hey sorry to bother you but it looks like you have a cracked version if im right.. i do so too. How did you managed to run mod engine on cracked version? i cant make it work whatever i do.

Literally all you need to do is extract the zip on here directly to your game folder. So like in OP's image, you'll see eldenring.exe and modengine2_launcher.exe in same folder.

**Comment by uYRO** [comment]
> Literally all you need to do is extract the zip on here directly to your game folder. So like in OP's image, you'll see eldenring.exe and modengine2_launcher.exe in same folder. Do not run the bat file, rather the launcher.exe

Didn't work :(

**Comment by F-annaray** [comment]
*thx for yr personal attention.*! :)
I can run the game as admin with the new bat file
I extracted the  mod HoarahLouxTrio.rar or LetMeSoloHerGuy.rar in the mod
folder of the mod engine 2 as instructed by its readme file. ( one or the
other, not both.)
In game it should replace the lone wolf ashes, but both of them did not. I
got the original wolves
What can i do?

High regards
:)
Kyrill

**Comment by AndroIDDQD** [comment]
> Установите этот флажок, если вы используете защитник Windows
> 
> ![изображение](https://user-images.githubusercontent.com/104241228/164866486-4af8f3a5-e6d1-415d-ac48-640b322fb7bb.png)

Работает ли ModEngine2 с НЕ-Стим версией игры или нет???
